### ISSUE TITLE
Not work or I'm doing wrong?

### ISSUE THREAD

**Issue by jadsonlourenco** [exponentjs/ex-navigator#3]
Well, I follow the readme, but get this error:

```
undefined is not an object (evaluating \'this._navigator.navigationContext\')'
```

I'm using RN 0.11.4, with ES6/ES7 features (like the readme say).
Thanks!

**Comment by jadsonlourenco** [comment]
Ok, with the pull #2 solve the first error, but get a new:

```
undefined is not an object (evaluating \'element.ref\')'
// cloneReferencedElement.js:7
```

**Comment by jesseruder** [comment]
Hey @jadsonlourenco sorry about that. I just pushed a fix for that error in #2

**Comment by jadsonlourenco** [comment]
Hey @jeremyschoenherr thank you! Now works fine, I use you last commit (9a790e3ce45ba3937d0e806f8e81213ab6b7788a).
